### PR TITLE
Upgrade to go 1.18.5, golangci-lint 1.47.3

### PR DESCRIPTION
**Pull request by petemoore**
Updates performed following [instructions](https://github.com/taskcluster/taskcluster/blob/main/dev-docs/node-and-go-upgrades.md#updating-go).

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3488,7 +3488,7 @@ hashids@^2.0.0:
   resolved "https://registry.yarnpkg.com/hashids/-/hashids-2.2.10.tgz#82f45538cf03ce73e31b78d1abe78d287cf760c4"
   integrity sha512-nXnYums7F8B5Y+GSThutLPlKMaamW1yjWNZVt0WModiJfdjaDZHnhYTWblS+h1OoBx3yjwiBwxldPP3nIbFSSA==
 
-hawk@9.0.1:
+hawk@9.0.1, hawk@^9.0.1:
```
Comment (petemoore) [inline yarn.lock:3491]: 🤯

File: yarn.lock [hunk 2]
```diff
@@ -6017,6 +6017,13 @@ slugid@^2.0.0:
     uuid "^3.2.1"
     uuid-parse "^1.0.0"
 
+slugid@^3.0.0:
+  version "3.0.0"
+  resolved "https://registry.yarnpkg.com/slugid/-/slugid-3.0.0.tgz#57cb692641a20feb4a1d774b2ad11594c9aa6125"
+  integrity sha512-MIir9SJS6BQFjFg8DHGMgPkDg21huT0+kYHiS6NkkWZXHSEdgIECo3nUVEG3GDkmecBTy+zRs6pCvPhpk4bKcg==
+  dependencies:
+    uuid "^8.3.2"
+
```
Comment (petemoore) [inline yarn.lock:6026]: Not sure why this wasn't there before, and why it is is there now.

File: yarn.lock [hunk 3]
```diff
@@ -6286,39 +6293,51 @@ tar-stream@^2.0.0:
 
 "taskcluster-client@link:clients/client":
   version "0.0.0"
+  uid ""
 
 "taskcluster-db@link:db":
-  version "44.17.0"
+  version "0.0.0"
+  uid ""
```
Comment (petemoore) [inline yarn.lock:6300]: WAT

File: yarn.lock [hunk 4]
```diff
@@ -6286,39 +6293,51 @@ tar-stream@^2.0.0:
...
 
 "taskcluster-lib-api@link:libraries/api":
-  version "44.17.0"
+  version "0.0.0"
+  uid ""
```
Comment (petemoore) [inline yarn.lock:6304]: WAT

File: yarn.lock [hunk 5]
```diff
@@ -6286,39 +6293,51 @@ tar-stream@^2.0.0:
...
 
 "taskcluster-lib-app@link:libraries/app":
-  version "44.17.0"
+  version "0.0.0"
+  uid ""
```
Comment (petemoore) [inline yarn.lock:6308]: WAT

File: yarn.lock [hunk 6]
```diff
@@ -6286,39 +6293,51 @@ tar-stream@^2.0.0:
...
 
 "taskcluster-lib-azqueue@link:libraries/azqueue":
-  version "44.17.0"
+  version "0.0.0"
+  uid ""
```
Comment (petemoore) [inline yarn.lock:6312]: WAT

File: yarn.lock [hunk 7]
```diff
@@ -6286,39 +6293,51 @@ tar-stream@^2.0.0:
...
 
 "taskcluster-lib-config@link:libraries/config":
-  version "44.17.0"
+  version "0.0.0"
+  uid ""
```
Comment (petemoore) [inline yarn.lock:6316]: WAT

File: yarn.lock [hunk 8]
```diff
@@ -6286,39 +6293,51 @@ tar-stream@^2.0.0:
...
 
 "taskcluster-lib-iterate@link:libraries/iterate":
-  version "44.17.0"
+  version "0.0.0"
+  uid ""
```
Comment (petemoore) [inline yarn.lock:6320]: WAT

File: yarn.lock [hunk 9]
```diff
@@ -6286,39 +6293,51 @@ tar-stream@^2.0.0:
...
 
 "taskcluster-lib-loader@link:libraries/loader":
-  version "44.17.0"
+  version "0.0.0"
+  uid ""
```
Comment (petemoore) [inline yarn.lock:6324]: WAT

File: yarn.lock [hunk 10]
```diff
@@ -6286,39 +6293,51 @@ tar-stream@^2.0.0:
...
 
 "taskcluster-lib-monitor@link:libraries/monitor":
-  version "44.17.0"
+  version "0.0.0"
+  uid ""
```
Comment (petemoore) [inline yarn.lock:6328]: WAT

File: yarn.lock [hunk 11]
```diff
@@ -6286,39 +6293,51 @@ tar-stream@^2.0.0:
...
 
 "taskcluster-lib-postgres@link:libraries/postgres":
-  version "44.17.0"
+  version "0.0.0"
+  uid ""
```
Comment (petemoore) [inline yarn.lock:6332]: WAT

File: yarn.lock [hunk 12]
```diff
@@ -6286,39 +6293,51 @@ tar-stream@^2.0.0:
...
 
 "taskcluster-lib-pulse@link:libraries/pulse":
-  version "44.17.0"
+  version "0.0.0"
+  uid ""
```
Comment (petemoore) [inline yarn.lock:6336]: WAT

File: yarn.lock [hunk 13]
```diff
@@ -6286,39 +6293,51 @@ tar-stream@^2.0.0:
...
 
 "taskcluster-lib-references@link:libraries/references":
-  version "44.17.0"
+  version "0.0.0"
+  uid ""
```
Comment (petemoore) [inline yarn.lock:6340]: WAT

File: yarn.lock [hunk 14]
```diff
@@ -6328,15 +6347,17 @@ taskcluster-lib-scopes@^11.0.0:
     fast-json-stable-stringify "^2.1.0"
 
 "taskcluster-lib-testing@link:libraries/testing":
-  version "44.17.0"
+  version "0.0.0"
+  uid ""
```
Comment (petemoore) [inline yarn.lock:6351]: WAT

File: yarn.lock [hunk 15]
```diff
@@ -6328,15 +6347,17 @@ taskcluster-lib-scopes@^11.0.0:
...
 
 taskcluster-lib-urls@^13.0.0:
   version "13.0.1"
   resolved "https://registry.yarnpkg.com/taskcluster-lib-urls/-/taskcluster-lib-urls-13.0.1.tgz#773da8bb9937a52f77ad128961b8d501dfdd042a"
   integrity sha512-WrhKMbiWmpPrB0vbzLZq4W35mhdPVLklZY+qrBoI7KQzFAjO/qsgS8ssHL6eYmvBf4fE7C+C9ZFxQpQOUynQvg==
 
 "taskcluster-lib-validate@link:libraries/validate":
-  version "44.17.0"
+  version "0.0.0"
+  uid ""
```
Comment (petemoore) [inline yarn.lock:6360]: WAT

File: yarn.lock (hunk 3 above)
Comment (lotas) [reply yarn.lock:6300]: this happens sometimes.. release task fixes it back 🤷

**Review by matt-boris — APPROVED** [review]
Nice! Ty!

**Pull request merged**